layouts/partials/article/components/details.html

```diff
@@ -34,7 +34,7 @@ <h3 class="article-subtitle">
             </div>
         {{ end }}
 
-        {{ if .Site.Params.article.readingTime }}
+    {{ if and (not (.Params.DisableReadTime | default false)) (.Site.Params.article.readingTime)}}
```

Comment:
I think it would be better to name the frontmatter field as `readingTime` (the same as in the configuration file), and set the default value of it to `.Site.Params.article.readingTime`.

Reply:
sounds good let me fix then

Reply:
Done!